cmake/TorchConfig.cmake.in

```diff
@@ -41,19 +41,37 @@ if (@BUILD_SHARED_LIBS@)
 endif()
 
 if (NOT ANDROID)
-  find_library(TORCH_LIBRARY torch PATHS "${TORCH_INSTALL_PREFIX}/lib")
+  if (NOT WIN32)
+    find_library(TORCH_LIBRARY torch PATHS "${TORCH_INSTALL_PREFIX}/lib")
+  else()
+    find_library(TORCH_LIBRARY_DEBUG torchd PATHS "${TORCH_INSTALL_PREFIX}/lib")
+    find_library(TORCH_LIBRARY_RELEASE torch PATHS "${TORCH_INSTALL_PREFIX}/lib")
+  endif()
 else()
   find_library(TORCH_LIBRARY NO_CMAKE_FIND_ROOT_PATH torch PATHS "${TORCH_INSTALL_PREFIX}/lib")
 endif()
 add_library(torch UNKNOWN IMPORTED)
-set(TORCH_LIBRARIES torch ${Caffe2_MAIN_LIBS})
+if (NOT WIN32)
+  set(TORCH_LIBRARIES torch ${Caffe2_MAIN_LIBS})
+else()
+  set(TORCH_LIBRARIES ${TORCH_LIBRARY_RELEASE} ${TORCH_LIBRARY_DEBUG} ${Caffe2_MAIN_LIBS})
```

Comment:
I don't think linking against both the debug and release libraries will work here. You should use `select_library_configurations` as an alternative.